packages/ref/conftest.py

```diff
@@ -54,9 +55,35 @@ def db_seeded_template(tmp_path_session, cmip6_data_catalog) -> Path:
     return template_db_path
 
 
+@pytest.fixture(scope="session")
+def db_seeded_template_obs4mips(tmp_path_session, obs4mips_data_catalog) -> Path:
```

Comment:
Can we just merge this with the above `db_seeded_template` unless you need a seeded db with just obs4mips data.